.travis.yml

```diff
@@ -45,12 +45,12 @@ jobs:
         - go get .
       script:
         - ./_tools/install-lookout-latest.sh
-        - (go run language-analyzer.go |& tee -a ../go-analyzer.log)&
+        - (go run language-analyzer.go |& tee -a ./go-analyzer.log)&
         - sleep 1s
-        - ../lookout_sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2020"
+        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2020"
       after_failure:
-        - cat ../go-analyzer.log
-        - cat ../lookout-install.log
+        - cat ./go-analyzer.log
```

Comment:
we would need to also gitignore all those files, in order for them not to mess everually with `no-changes-commit`

Reply:
done in fd07f5e